packages/sources/css/src/components/card/src/index.css

```diff
@@ -122,13 +119,13 @@
   height: auto;
   border-radius: 8px;
   color: var(--vtmn-semantic-color_content-primary-reversed);
-  background-color: linear-gradient(
+  background: linear-gradient(
```

Comment:
why not keeping `background-color` in this case?

Reply:
Because it bugs with the previous background-color, idk why